components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIGatewayManager.java

```diff
@@ -145,6 +148,9 @@ public synchronized static APIGatewayManager getInstance() {
         if (api.getGatewayLabels() != null) {
             for (Label label : api.getGatewayLabels()) {
                 Environment environment = getEnvironmentFromLabel(label);
+                if (debugEnabled) {
+                    log.debug(api.getId() + " publishing to the environment of " + environment);
```

Comment:
Since this is a label, change the debug message to label. Change the environment object to environment name

Reply:
https://github.com/wso2/carbon-apimgt/pull/9019/commits/29a2e4bba591e33476f9fcce64bd8668945123c4